site.properties

```diff
@@ -44,25 +44,31 @@ mail.username = user@gmail.com
 mail.password = password
 
 # ETL scheduler pipeline
-etlplugin.pipeline.s3.bucketname =
-etlplugin.pipeline.s3.region =
-airflow.username =
-airflow.password =
-airflow.hostname =
-etl.s3.url =
+etlplugin.pipeline.s3.bucketname = mayank-test-databucket
+etlplugin.pipeline.s3.region = ap-southeast-2
+airflow.username = airflow
+airflow.password = airflow
+airflow.hostname = http://localhost:8080
+etl.s3.url = S3::mayank-test-databucket::ap-southeast-2::https://s3.ap-southeast-2.amazonaws.com
+apromore.tenant = ap
+
+# Impala
+impala.jdbc.url      = jdbc:impala://localhost:21050/
+impala.jdbc.username = apromore
+impala.jdbc.password = MAcri
+impala.etl.dataStore = /etl_data
+impala.etl.s3.directory = users/
```

Comment:
These properties don't look like they belong in the open source default configuration.